src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -152,10 +152,6 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\BodyAndPath-LowLevel\\Generated"
     },
-    "CadlFirstTest": {
```

Comment:
Please don't kill this project yet, it is usable as a playground.

Reply:
@chunyu3 plans to remove it. Do we have a substitute one?